modules/microshift-updating-pods-mismatch.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/understanding-persistent-storage-microshift.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-updating-pods-mismatch_{context}"]
+= Updating the pods which have mismatch
+
+Update the SELinux context of the pods if a mismatch is found by using the following procedure.
+
+.Procedure
+
+. When there is a mismatch of the SELinux content, create a new security context constraint (SCC) and assign it to both pods. To create a SCC, see link:https://docs.openshift.com/container-platform/4.15/authentication/managing-security-context-constraints.html#security-context-constraints-creating_configuring-internal-oauth[Creating security context constraints].
```

Comment:
I think I understand why you used a `link` instead of an `xref`, since it looks like the format of the Creating SCC page has changed significantly since the 4.15 release. 

I will merge this as-is, but wanted to make a note of it. If you change your mind later, you can always revert to an `xref`.

Reply:
Using "Link" here and not xref  is right as the link take us outside of MicroShift page. MicroShift uses a separate topic map and xrefs do not work across topic maps.